src/MSBuild.Conversion.Facts/MSBuildFacts.cs

```diff
@@ -246,10 +246,12 @@ public static class MSBuildFacts
             "ProjectReference"
         );
 
+        public static Guid LanguageProjectTypeVisualBasic => Guid.Parse("{F184B08F-C81C-45F6-A57F-5ABD9991F28F}");
```

Comment:
This is the guid that was labeled F# previously. What that comment wrong or do we need to swap this guid?

Reply:
I believe the comment was wrong.

Reply:
if I recall, this was pulled from a legacy F# project. Interesting!